lib/msf/core/exploit/remote/kerberos/client/tgs_request.rb

```diff
@@ -284,6 +287,41 @@ def build_tgs_body_checksum(opts = {})
 
               checksum
             end
+
+            def build_pa_for_user(opts = {})
```

Comment:
Missing yard docs 👀 

And an extra link to `https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-sfu/aceb70de-40f0-4409-87fa-df00ca145f5a` would be good 🎉

Reply:
Good catch, I forgot this one. Thanks!